addons/func_godot/src/trenchbroom/trenchbroom_game_config.gd

```diff
@@ -43,14 +50,21 @@ extends Resource
 @export_category("Editor Hint Tags")
 
 ## TrenchBroomTag resources that apply to brush entities.
-@export var brush_tags : Array[Resource] = []
+@export var brush_tags : Array[Resource] = [
+	preload("res://addons/func_godot/game_config/trenchbroom/tb_brush_tag_func.tres"),
+	preload("res://addons/func_godot/game_config/trenchbroom/tb_brush_tag_trigger.tres")
+]
```

Comment:
Brush tags should be empty, no default brush tags. We don't assume a user will be using `func_` or `trigger_` prefixes for their entities.